sim/common/vanilla/item_effects.go

```diff
@@ -1836,11 +1837,13 @@ func init() {
 		})
 	})
 
+	itemhelpers.CreateWeaponCoHProcDamage(TheNeedler, "The Needler", 3.0, 13060, core.SpellSchoolPhysical, 75, 0, 0, core.DefenseTypeMelee)
```

Comment:
This one looks slightly out of order since `The Jackhammer` would come first (sorry it's super nit picky I just use the ordering to find what item I'm looking for a lot 😅)

Reply:
fixed